integration-tests/src/test/java/gov/uk/di/ipv/cri/common/api/stepDefinitions/APISteps.java

```diff
@@ -1,27 +1,35 @@
 package gov.uk.di.ipv.cri.common.api.stepDefinitions;
 
 import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import gov.uk.di.ipv.cri.common.api.util.IpvCoreStubUtil;
 import io.cucumber.java.en.And;
 import io.cucumber.java.en.Given;
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
+import org.apache.http.NameValuePair;
+import org.apache.http.message.BasicNameValuePair;
 
 import java.io.IOException;
 import java.net.URISyntaxException;
 import java.net.http.HttpResponse;
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Map;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 
 public class APISteps {
 
-    private final String DEV_SESSION_URI = "/dev/session";
+    private final String ENVIRONMENT = "/dev"; // dev, build, staging, integration
+    private final String DEV_SESSION_URI = ENVIRONMENT + "/session";
+    private final String DEV_AUTHORIZATION_URI = ENVIRONMENT + "/authorization";
+    private final ObjectMapper objectMapper = new ObjectMapper();
     private String sessionRequestBody;
+    private String sessionId;
```

Comment:
It may be worth renaming `sessionId` to `currentSessionId` to provide a bit more context